euth/documents/validators.py

```diff
@@ -1,5 +1,4 @@
 from django.core.exceptions import ValidationError
-
```

Comment:
You commit a lot of those changes in commits that try to implement a specific feature. I guess because your run isort. While this also does no harm it is cleaner to put this in an extra commit, or even extra branch. Because right now your changes to the user profile touched files in all components.